lib/child-writer/writer-proxy.js

```diff
@@ -176,11 +178,6 @@ permissions.isElevated().then((elevated) => {
       ipc.of[process.env.IPC_SERVER_ID].on('connect', () => {
         // Inherit the parent evnironment
         const childEnv = _.assign({}, process.env, {
```

Comment:
How come this still uses the "bulk" `process.env`, rather than only copying out specific keys as the change on line 104 does?

Reply:
Because this is a "normal" child process spawn, whereas the elevation prompt gets the env prepended to the command, which causes the issue as mentioned in the description – hence we can just pump the entire env through here.

Reply:
Right, read that a couple of times and now I understand ;-)
So we _can_ just pump the entire env through here, but _should_ we? Would it be "better" to only pass in the env-vars that we know we'll need?